DSCResources/CommonResourceHelper.psm1

```diff
@@ -0,0 +1,142 @@
+﻿<#
+    .SYNOPSIS
+    Tests if the current machine is a Nano server.
+#>
+function Test-IsNanoServer
+{
+    [OutputType([Boolean])]
+    [CmdletBinding()]
+    param ()
+
+    return $PSVersionTable.PSEdition -ieq 'Core'
```

Comment:
@TravisEz13 - sorry - I found where you'd put the correct code (I hadn't got to your other code review yet). I'll raise an issue against the PSDscResources and xPSDesiredStateConfiguration repos to get these updated to use the correct method.